src/web_main/settings.py

```diff
@@ -30,8 +30,10 @@
 
 ALLOWED_HOSTS = ["stac.iitmandi.co.in", "127.0.0.1", "localhost"]
 
-DEBUG = False
-
+try:
```

Comment:
This try-error block should be avoided, we do not use os in production. An alternative is:
```
try:
    import secret
    SECRET_KEY = secret.SECRET_KEY
    DEBUG = False
except ImportError:
    SECRET_KEY = os.environ.get("SECRET_KEY")
    DEBUG = True
```

Reply:
Ok. Will do that